### PR TITLE
Update LSP-1-UniversalReceiver.md

### PR DESCRIPTION
**Pull request by KiriKev**
Proactive improvement proposal:
- Corrected grammar, misspelling and punctuation marks
- Changed 'should' to 'SHOULD'
- Exported hyperlinks to the bottom of the document aka the list of references for a cleaner code
- Changed wording slightly to improve comprehension
- Inserted paragraphs to improve readability
- Split too long sentences to improve comprehension and readability

### CODE REVIEW

**Review by YamenMerhi — APPROVED** [review]
🚀

**Review by b00ste — APPROVED** [review]

File: LSPs/LSP-1-UniversalReceiver.md [hunk 1]
```diff
@@ -44,18 +46,20 @@ Every contract that complies with the LSP1-UniversalReceiver standard MUST imple
 function universalReceiver(bytes32 typeId, bytes memory data) external payable returns (bytes memory)
 ```
 
-Allows to be called by any external contract to inform the contract about any incoming transfers, interactions or simple information. Can be customized to react on the different aspect of the call such as the typeId, the data sent, the caller or the value sent to the function, e.g, reacting on a token or a vault transfer. 
+Allows to be called by any external contract to inform the contract about any incoming transfers, interactions or simple information. 
+
+The `universalReceiver(...)` function can be customized to react on a different aspect of the call such as the `typeId`, the data sent, the caller or the value sent to the function, e.g, reacting on a token or a vault transfer.
```
Comment (CJ42) [inline LSPs/LSP-1-UniversalReceiver.md:51]: I suggest to example text in italic.

```suggestion
The `universalReceiver(...)` function can be customized to react on a different aspect of the call such as the `typeId`, the data sent, the caller or the value sent to the function (_e.g, reacting on a token or a vault transfer_).
```

File: LSPs/LSP-1-UniversalReceiver.md [hunk 2]
```diff
@@ -86,7 +90,9 @@ UniversalReceiver delegation allows to forward the `universalReceiver(..)` call
 
 ### Motivation
 
-The ability to react to upcoming actions with a logic hardcoded within the `universalReceiver(..)` function comes with a limitations, as only a fixed functionality can be coded or the `UniversalReceiver` be fired. This section explains a way to forward the call to the `universalReceiver(..)` function to an external smart contract to extend and change funcitonality over time.
+The ability to react to upcoming actions with a logic hardcoded within the `universalReceiver(..)` function comes with limitations, as only a fixed functionality can be coded or the `UniversalReceiver` be fired.
```
Comment (CJ42) [inline LSPs/LSP-1-UniversalReceiver.md:93]: Can link to the `UniversalReceiver` event section here :)

```suggestion
The ability to react to upcoming actions with a logic hardcoded within the `universalReceiver(..)` function comes with limitations, as only a fixed functionality can be coded or the [`UniversalReceiver`](#universalreceiver-1) event be fired. 
```

File: LSPs/LSP-1-UniversalReceiver.md [hunk 3]
```diff
@@ -101,19 +107,27 @@ The `universalReceiver(..)` function on the initial smart contract forwards the
...
+The **UniversalReceiverDelegate** smart contract can then understand the `msg.sender` and `msg.value` of the initial smart contract, or ignore the appended data.
 
 
 ## Rationale
-This is an abstraction of the ideas behind Ethereum [ERC223](https://github.com/ethereum/EIPs/issues/223) and [ERC777](https://eips.ethereum.org/EIPS/eip-777), that contracts are called when they are receiving tokens or other assets. With this proposal, we can allow contracts to receive any information over a standardised interface. As this function is generic and only the send `typeId` changes, smart contract accounts that can upgrade its behaviour using the **UniversalReceiverDelegate** technique can be created. UniversalReceiverDelegate functionality COULD be implemented using `call`, or `delegatecall`, both of which have different security properties.
+This is an abstraction of the ideas behind [ERC223] and [ERC777], that contracts are called when they are receiving tokens or other assets. 
+
+With this proposal, we can allow contracts to receive any information over a standardised interface.
```
Comment (CJ42) [inline LSPs/LSP-1-UniversalReceiver.md:116]: Can you add this extra point please? :)

```suggestion
With this proposal, we can allow contracts to receive any information in a generic manner over a standardised interface. 
```

**Review by CJ42 — APPROVED** [review]
Amazing thanks a lot @KiriKev for these improvements!

I have added few small comments.

**Pull request merged**